server/src/controllers/Auth.controller.js

```diff
@@ -68,4 +69,59 @@ export default class AuthController {
       message: 'User registered successfully'
     });
   }
+
+  /**
+   * @method login
+   *
+   * @param {object} req request
+   * @param {object} res response
+   *
+   * @returns {object} returns status code, data and message properties
+   */
+  static login(req, res) {
+    const { email, password } = req.body;
+
+    for (let i = 0; i < users.length; i++) {
+      if (email === users[i].email) {
+        const verifyPassword = comparePassword(password, users[i].password);
+
+        if (!verifyPassword) {
+          return res.status(404).json({
+            status: 404,
+            error: 'Email or password is incorrect'
+          });
+        }
+        const userInfo = users[i];
+
+        const payload = {
+          id: userInfo.id,
+          firstName: userInfo.firstName,
+          lastName: userInfo.lastName,
+          email: userInfo.email,
+          type: userInfo.type,
+          isAdmin: userInfo.isAdmin
+        };
+
+        const token = jwt.sign(payload, process.env.SECRET_KEY, { expiresIn: '1day' });
+
+        const data = {
+          token,
+          id: userInfo.id,
+          firstName: userInfo.firstName,
+          lastName: userInfo.lastName,
+          email: userInfo.email,
+          type: userInfo.type
```

Comment:
Really great job not exposing the password

Reply:
thanks